src/lightning_app/cli/lightning_cli.py

```diff
@@ -491,7 +514,12 @@ def init_component(name):
 
 
 @init.command("react-ui")
-@click.option("--dest_dir", "-dest_dir", type=str, help="optional destination directory to create the react ui")
+@click.option(
+    "--dest_dir",
+    "-dest_dir",
```

Comment:
that is strange, single `-` normally have just one letter

Reply:
strange, this is not intentional probably formatter.